### PR TITLE
Laravel volumized storage section

### PR DESCRIPTION
**Pull request by KTanAug21**
Add section for mounting volume to the storage folder, which requires reinitialization after volume mount

### CODE REVIEW

**Comment by fideloper** [comment]
Good use of partials!

File: laravel/partials/_volume_storage_steps.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+1.  Make sure you are in your Laravel Fly App's directory
+
+    ```cmd
```
Comment (fideloper) [inline laravel/partials/_volume_storage_steps.html.md.erb:3]: Just double check the formatting when using both a tab (err 4 spaces or whatever) AND the back-tics for code blocks.

**Review by fideloper — APPROVED** [review]

File: laravel/partials/_volume_storage_steps.html.md.erb (hunk 1 above)
Comment (KTanAug21) [reply laravel/partials/_volume_storage_steps.html.md.erb:3]: yes sir! checking this one

Comment (KTanAug21) [reply laravel/partials/_volume_storage_steps.html.md.erb:3]: update: 
I think the tabs help keep the numbering in the list,

 with tabs:
image

___________________________________________________________________________
but removing the tabs/spaces restart the numbering of the items: to 1,1, 
image

so probably tabs group sections under an item

**Pull request merged**